src/Apps/Artwork/Utils/createCollectUrl.tsx

```diff
@@ -0,0 +1,60 @@
+import * as qs from "qs"
+import { data as sd } from "sharify"
+
+export const createCollectUrl = ({
+  dimension,
+  category,
+  sizeScore,
+  artistId,
+}) => {
+  let dimensionRange
+
+  if (dimension === null) {
+    dimensionRange = "*-*"
+  } else if (sizeScore < 1600) {
+    dimensionRange = "0-1600"
+  } else if (1600 < sizeScore && sizeScore < 4900) {
+    dimensionRange = "1600-4900"
+  } else {
+    dimensionRange = "4900-*"
+  }
+
+  const query = qs.stringify({
+    page: 1,
+    sort: "-decayed_merch",
+    acquireable: "true",
+    offerable: "true",
+    inquireable_only: "true",
+    dimension_range: dimensionRange,
+    artist_id: artistId,
+  })
+
+  const path = [sd.APP_URL, "collect", filterCategories[category]]
+    .filter(Boolean)
+    .join("/")
+
+  return `${path}?${query}`
+}
+
+// these come from MediumFilter.tsx
+const filterCategories = {
+  Architecture: "",
+  "Books and Portfolios": "",
+  "Design/Decorative Art": "design",
+  "Drawing, Collage or other Work on Paper": "drawing",
+  "Fashion Design and Wearable Art": "wearable art",
```

Comment:
where did `"wearable art"` come from? I don't see it as an option in `MediumFilter.tsx`

Reply:
Idk I pulled these from the previous createCollectUrl for the clickable bars

Reply:
haha turns out I added this :joy: https://github.com/artsy/reaction/commit/fec57ec0a8a2f7b50a3f7c234e28208dca6de533#diff-a829f088eac28b6207af9063873fc2ec

Let's change `"wearable art"` to `""` for now. `"wearable art"` is not a thing that is handled anywhere, and there doesn't seem to be an analogous filter.